.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "rust-g"]
+	path = rust-g
+	url = https://github.com/tgstation/rust-g
```

Comment:
Can we not do this again please? Last time literally everyone was coming to us with broken gits/prs.